<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-08-16 09:18:04.825 ***</b>
=ERROR REPORT==== 16-Aug-2024::09:18:04.824920 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.244181.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1805}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1766}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2024-08-16 09:18:04.825 ***</b>
=CRASH REPORT==== 16-Aug-2024::09:18:04.825465 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.244189.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2254)
    ancestors: [<0.244181.0>]
    message_queue_len: 0
    messages: []
    links: [<0.244181.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 6882
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-08-16 09:18:43.395 ***</b>
=ERROR REPORT==== 16-Aug-2024::09:18:43.395362 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.248533.0&gt;,
                               {#Ref&lt;0.651655849.1369964545.125817&gt;,1006062,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.651655849.1369964545.125817&gt;,1006062,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2024-08-16 09:18:43.396 ***</b>
=CRASH REPORT==== 16-Aug-2024::09:18:43.395830 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.248554.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.651655849.1369964545.125817&gt;,1006062,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [&lt;0.248533.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 4752
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2024-08-16 09:23:21.773 ***</b>
=CRASH REPORT==== 16-Aug-2024::09:23:21.773500 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.254296.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.651655849.1371275265.248248&gt;,6334,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1522)
    ancestors: [&lt;0.254294.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 28672
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-08-16 09:23:31.513 ***</b>
=ERROR REPORT==== 16-Aug-2024::09:23:31.511860 ===
** Generic server <0.255591.0> terminating 
** Last message in was {nodedown,'peer_down_crash-383973-11794@7326fd391957'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-383973-11794",
                              peer_down => crash},
                            'peer_down_crash-383973-11794@7326fd391957',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,48,46,49,47,98,
                             105,110,47,101,114,108],
                            ["-sname","peer_down_crash-383973-11794",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQDczMjZmZDM5MTk1NwAD5mcAAAAAZr76yw=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-383973-11794@7326fd391957'}},
                            {<0.255563.0>,#Ref<0.651655849.1363935236.31397>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-383973-11794@7326fd391957'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2024-08-16 09:23:31.515 ***</b>
=CRASH REPORT==== 16-Aug-2024::09:23:31.514871 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255591.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-383973-11794@7326fd391957'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2304)
    ancestors: [<0.255563.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11199
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-08-16 09:23:32.550 ***</b>
=WARNING REPORT==== 16-Aug-2024::09:23:31.549151 ===
Process <0.91.0> on node 'peer_SUITE-shutdown_stop-530722-11794@7326fd391957' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-08-16 09:23:32.984 ***</b>
=WARNING REPORT==== 16-Aug-2024::09:23:32.983901 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-530850-11794@7326fd391957'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2024-08-16 09:23:33.072 ***</b>
=WARNING REPORT==== 16-Aug-2024::09:23:33.072847 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-384037-11794@7326fd391957'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-08-16 09:23:49.441 ***</b>
=WARNING REPORT==== 16-Aug-2024::09:23:48.440336 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-397060-11794@7326fd391957' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-08-16 09:23:49.959 ***</b>
=WARNING REPORT==== 16-Aug-2024::09:23:49.958873 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-397028-11794@7326fd391957'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2024-08-16 09:23:50.005 ***</b>
=WARNING REPORT==== 16-Aug-2024::09:23:50.005799 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-384261-11794@7326fd391957'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 09:23:52.860 ***</b>
=ERROR REPORT==== 16-Aug-2024::09:23:52.860309 ===
** Generic server <0.255924.0> terminating 
** Last message in was {'EXIT',<0.255920.0>,
                               {#Ref<0.651655849.1364197378.19495>,1756232,ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-442179-11794",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-442179-11794@7326fd391957',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,48,46,49,47,98,105,110,47,101,114,
                                     108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-442179-11794",
                                     "-user","peer"],
                                    #Port<0.3114>,undefined,<<>>,running,
                                    {<0.255920.0>,
                                     #Ref<0.651655849.1364197378.19608>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.651655849.1364197378.19495>,1756232,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 09:23:52.861 ***</b>
=CRASH REPORT==== 16-Aug-2024::09:23:52.860879 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255924.0>
    registered_name: []
    exception exit: {#Ref<0.651655849.1364197378.19495>,1756232,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.255920.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3114>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 13985
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 09:23:53.836 ***</b>
=WARNING REPORT==== 16-Aug-2024::09:23:52.835129 ===
Process <0.88.0> on node 'peer_SUITE-shutdown_stop-384613-11794@7326fd391957' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 09:23:54.050 ***</b>
=WARNING REPORT==== 16-Aug-2024::09:23:54.050824 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-397348-11794@7326fd391957'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 09:23:54.179 ***</b>
=WARNING REPORT==== 16-Aug-2024::09:23:54.179864 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-442211-11794@7326fd391957'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 09:23:54.249 ***</b>
=WARNING REPORT==== 16-Aug-2024::09:23:54.249819 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-384677-11794@7326fd391957'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 09:23:55.327 ***</b>
=ERROR REPORT==== 16-Aug-2024::09:23:55.327583 ===
** Generic server <0.255967.0> terminating 
** Last message in was {'EXIT',<0.255909.0>,
                           {#Ref<0.651655849.1363935235.208698>,4225171,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "PTTUCVCWOXLEWMUQOBSH","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@7326fd391957.2024-08-16_09.07.54/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-397412-11794",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@7326fd391957.2024-08-16_09.07.54/make_test_dir.stdlib_test.logs/run.2024-08-16_09.08.39/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-397412-11794",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-25.3.2.13/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-397412-11794@7326fd391957',
                                    "/usr/local/lib/erlang-25.3.2.13/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-397412-11794",
                                     "-setcookie","PTTUCVCWOXLEWMUQOBSH",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@7326fd391957.2024-08-16_09.07.54/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-397412-11794",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@7326fd391957.2024-08-16_09.07.54/make_test_dir.stdlib_test.logs/run.2024-08-16_09.08.39/log_private/",
                                     "-user","peer"],
                                    #Port<0.3142>,undefined,<<>>,running,
                                    {<0.255909.0>,
                                     #Ref<0.651655849.1364197378.21587>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.651655849.1363935235.208698>,4225171,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2024-08-16 09:23:55.328 ***</b>
=CRASH REPORT==== 16-Aug-2024::09:23:55.328509 ===
  crasher:
    initial call: peer:init/1
    pid: <0.255967.0>
    registered_name: []
    exception exit: {#Ref<0.651655849.1363935235.208698>,4225171,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.255909.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3142>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 17731
    stack_size: 29
    reductions: 18965
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-08-16 09:34:41.966 ***</b>
=ERROR REPORT==== 16-Aug-2024::09:34:41.966201 ===
** Generic server <0.302652.0> terminating 
** Last message in was {'EXIT',<0.302650.0>,
                               {#Ref<0.651655849.1401421825.37628>,351066,
                                [{child1,<0.302659.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302652.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302658.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459145],
                            0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.651655849.1401421825.37628>,351066,
    [{child1,<0.302659.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-08-16 09:34:41.966 ***</b>
=ERROR REPORT==== 16-Aug-2024::09:34:41.966201 ===
** Generic server <0.302653.0> terminating 
** Last message in was {'EXIT',<0.302650.0>,
                               {#Ref<0.651655849.1401421825.37628>,351066,
                                [{child1,<0.302659.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.302653.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.302659.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576459145],
                            0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.651655849.1401421825.37628>,351066,
    [{child1,<0.302659.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-08-16 09:34:41.966 ***</b>
=CRASH REPORT==== 16-Aug-2024::09:34:41.966754 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302652.0>
    registered_name: []
    exception exit: {#Ref<0.651655849.1401421825.37628>,351066,
                     [{child1,<0.302659.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.302650.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11913
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2024-08-16 09:34:41.967 ***</b>
=CRASH REPORT==== 16-Aug-2024::09:34:41.967029 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.302653.0>
    registered_name: []
    exception exit: {#Ref<0.651655849.1401421825.37628>,351066,
                     [{child1,<0.302659.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2127)
    ancestors: [<0.302650.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 4185
    stack_size: 29
    reductions: 11956
  neighbours:


</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2024 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Fri Aug 16 2024 09:08:39<!--/date--><br />
</div>
</center>
</body>
</html>
